pkg/resource/feature_group/testdata/feature_group/cr/fg_cmd_invalid_before_create.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: FeatureGroup
+metadata:
+  name: unit-testing-feature-group
+spec:
+  eventTimeFeatureName: EventTime
+  featureDefinitions:
+  - featureName: TransactionID
+    featureType: Integral
+  - featureName: EventTime
+    featureType: Fractional
+  featureGroupName: !unit-testing-feature-group
+  offlineStoreConfig:
+    s3StorageConfig:
+      s3URI: s3://source-data-bucket-592697580195-us-west-2/sagemaker/feature-group-data
```

Comment:
is this public `592697580195` ?

Reply:
yes: https://github.com/aws-controllers-k8s/sagemaker-controller/blob/60bbd16ebd8ce39970f796f47ff26f128b5fae3d/test/e2e/bootstrap_resources.py#L21